config/commandlogger.php

```diff
@@ -19,6 +19,11 @@
 	 */
 	'db_table' => 'commands_log',
 
+	/*
+	 * Indicates if CommandLogger migrations will be run.
+	 */
+	'runs_migrations' => true,
```

Comment:
isn`t is better to pass env value here?

Reply:
so you will need to keep the value in the environment at all times